topics/metadata/assembly.yaml

```diff
@@ -1 +0,0 @@
-../topics/assembly/metadata.yaml
```

Comment:
Oh, was that deprecated recently? We need to remove it from the skeleton generator in that case.

Thanks for fixing.

Reply:
It wasn't removed, it seems like an accidental symlink was added. This was added to a topic called 'metadata' which doesn't exist.